pkgs/development/libraries/libva/default.nix

```diff
@@ -7,22 +7,22 @@
 
 stdenv.mkDerivation rec {
   name = "libva-${lib.optionalString minimal "minimal-"}${version}";
-  version = "2.6.1";
+  version = "2.7.1"; # Also update the hash for libva-utils!
 
   # update libva-utils and vaapiIntel as well
   src = fetchFromGitHub {
-    owner  = "01org";
+    owner  = "intel";
     repo   = "libva";
     rev    = version;
-    sha256 = "1x34kf38p5rf52bf54ljr9f7knnbilm7kbszqnfk3lzsqrfc7r2g";
+    sha256 = "0ywasac7z3hwggj8szp83sbxi2naa0a3amblx64y7i1hyyrn0csq";
   };
 
   outputs = [ "dev" "out" ];
 
   nativeBuildInputs = [ autoreconfHook pkgconfig wayland ];
 
   buildInputs = [ libdrm ]
-    ++ lib.optionals (!minimal) [ libva-minimal libX11 libXext libXfixes wayland libffi libGL ];
+    ++ lib.optionals (!minimal) [ libva-minimal libX11 libXext libXfixes libffi libGL ];
```

Comment:
I see this does depend on wayland optionally https://github.com/intel/libva/blob/master/meson.build#L99, and I don't think we should change this?

Reply:
Yes, we should definitely keep Wayland support. I only removed it because it's already in `nativeBuildInputs` (for `wayland-scanner`). IIRC most packages don't put dependencies in both `nativeBuildInputs` and `buildInputs` if they're needed at build- and run-time, but after quickly looking over the documentation this should probably be the case: "But since these packages are only guaranteed to be able to run then, they shouldn't persist as run-time dependencies. This isn't currently enforced, but could be in the future."

Do you know more about this? Anyway, I reverted the change to avoid potential problems ;)

Reply:
It links against it here https://github.com/intel/libva/blob/7eb84e41658944ab0713c5aa6b8eddbcb50a114d/va/meson.build#L239 (and below), so that makes it always a run-time dependency.

Reply:
But yes, @primeos, it should also be in `nativeBuildInputs` for `wayland-scanner`. 
It is both native and build.

Reply:
I was mainly wondering if it should be in `buildInputs` as well if it's a build- and run-time dependency (just in case you happen to know). IIRC many Nixpkgs expressions add it ONLY to `nativeBuildInputs`, which I though was correct, but after reading the documentation that seems to be wrong.

But in practice it currently shouldn't matter anyway AFAIK (I didn't check the code).

Reply:
>  was mainly wondering if it should be in buildInputs as well if it's a build- and run-time dependency (just in case you happen to know). IIRC many Nixpkgs expressions add it ONLY to nativeBuildInputs, which I though was correct, but after reading the documentation that seems to be wrong.

This is what I meant with 

> But yes, @primeos, it should also be in nativeBuildInputs for wayland-scanner.
It is both native and build.

wayland-scanner is used here
* https://github.com/intel/libva/blob/7eb84e41658944ab0713c5aa6b8eddbcb50a114d/va/meson.build#L228

that is to generate files. So that makes the dependency on `wayland` native because it's used to generate something during build, but doesn't need to persist at runtime.

But at https://github.com/intel/libva/blob/7eb84e41658944ab0713c5aa6b8eddbcb50a114d/va/meson.build#L239, we are using wayland to link against so it's needed at runtime, so a `buildInputs`.

(slightly simplified)

Reply:
> This is what I meant with

Ok, sorry, I did interpret it a bit differently. Thanks for your insights :)